doc/api/util.md

```diff
@@ -666,7 +666,7 @@ Deprecated predecessor of `console.log`.
 
 Deprecated predecessor of `console.log`.
 
-### util._extend(obj)
+### util._extend(targetObj, sourceObj)
```

Comment:
@yorkie In the case of `origin/add` pair, I have code in place to understand what the particular function does. But documentation is the face of the code and we can not expect user to read the code in case of doubt. Its my humble opinion.